scripts/pre_commit_hook.py

```diff
@@ -73,8 +74,19 @@ def install_hook():
         try:
             os.symlink(os.path.abspath(this_file), pre_commit_file)
             python_utils.PRINT('Created symlink in .git/hooks directory')
-        # Raises AttributeError on windows, OSError added as failsafe.
-        except (OSError, AttributeError):
+        # Raises OSError when the symlink already exists but is broken.
+        except OSError as e:
```

Comment:
There's a lot of duplication here.

How about the following alternative: why don't you write os.unlink() above line 74 (perhaps just have an "if it exists" check) and then keep the rest of the existing code as-is? Would that work?

Ditto for the pre-push hook.

Reply:
Good point -- makes sense! Pushing a new version soon with unit tests. I've been fixing lint errors for almost an hour now haha. Is there a way to run only the Python lint checks? Maybe even for only a specific file?